resallocserver/manager.py

```diff
@@ -47,8 +47,15 @@ def run_command(func, res_id, res_name, command, ltype='alloc'):
             raise
 
     lfile = os.path.join(ldir, '{0:06d}_{1}'.format(res_id, ltype))
-    with open(lfile, 'a+') as logfile:
-        rv = func(command, env=env, shell=True, stderr=logfile)
+    with open(lfile, 'a+b') as logfile:
+        logfile.write(b'--------begin--------\n')
+        logfile.write(b'-----std.err-----\n')
+        rv = subprocess.check_output(command, env=env, shell=True,
+                                     stderr=logfile)
+        logfile.write(b'\n-----std.out-----\n')
+        logfile.write(rv)
```

Comment:
Could we have (a) two files (one for stderr, second for stdout), or (b) have interleaved (a bit more readable) log?  I would also like to avoid the `---xyz---` sugar.